app/components/Poll/index.tsx

```diff
@@ -51,197 +51,243 @@ type OptionEntityRatio = OptionEntity & {
 const Poll = ({
   poll,
   allowedToViewHiddenResults,
-  truncate,
   details,
+  alwaysOpen = false,
 }: Props) => {
-  const [truncateOptions, setTruncateOptions] = useState(false);
-  const [expanded, setExpanded] = useState(true);
-
-  useEffect(() => {
-    const options = poll && optionsWithPerfectRatios(poll.options);
-
-    if (truncate && options && options.length > truncate) {
-      setTruncateOptions(true);
-      setExpanded(false);
-    } else {
-      setTruncateOptions(false);
-      setExpanded(true);
-    }
-  }, [poll, truncate]);
-
-  const toggleTruncate = () => {
+  const optionRef = useRef<HTMLDivElement>(null);
+  const [expanded, setExpanded] = useState(alwaysOpen);
+
+  const toggleExpanded = () => {
     setExpanded(!expanded);
   };
 
   const fetching = useAppSelector(
     (state) => state.frontpage.fetching || state.polls.fetching
   );
-  const dispatch = useAppDispatch();
 
-  const options = poll && optionsWithPerfectRatios(poll.options);
-  const orderedOptions = options;
-  const optionsToShow = expanded
-    ? orderedOptions
-    : orderedOptions?.slice(0, truncate);
-  const showResults = !poll?.resultsHidden || allowedToViewHiddenResults;
+  if (fetching) {
+    return <Skeleton className={styles.poll} />;
+  }
```

Comment:
```suggestion
  if (fetching && !poll) {
    return <Skeleton className={styles.poll} />;
  }
```

No reason to show skeleton component if we have a poll to show - imo.